### ISSUE TITLE
Get working with woorelease script

### ISSUE THREAD

**Issue by allendav** [woocommerce/woocommerce-gateway-payfast#17]
woorelease appears to flag all of the files for this plugin as missing/not under version control (not sure what that last file is there)

`wr --no-build --version=1.4.15 woocommerce-gateway-payfast`

...

WR [18:55:42] ------------------
!       trunk/assets
!       trunk/assets/images
!       trunk/assets/images/icon.png
!       trunk/changelog.txt
!       trunk/gateway-payfast.php
!       trunk/includes
!       trunk/includes/class-wc-gateway-payfast-privacy.php
!       trunk/includes/class-wc-gateway-payfast.php
!       trunk/readme.txt
?       trunk/woocommerce-gateway-payfast
WR [18:55:42] ------------------

**Comment by jeffpaul** [comment]
Will check and see if this occurs as a release is handled for https://github.com/woocommerce/woocommerce-gateway-payfast/pull/48.

**Comment by jeffpaul** [comment]
@LCmry has this been an issue in [recent PayFast releases](https://github.com/woocommerce/woocommerce-gateway-payfast/releases)?

**Comment by LCmry** [comment]
@jeffpaul no it hasn't been an issue with the recent releases. We are releasing PayFast soon, so maybe we'll keep this until after the release to make sure nothing is an issue and then we'll close it if not.

**Comment by leonardola** [comment]
@allendav the command you posted does not seem to execute anything.
Here's what I see after running it:
<img width="395" alt="Screenshot 2022-12-06 at 3 36 53 PM" src="https://user-images.githubusercontent.com/6342964/205994000-d94cd519-c4c2-41b0-87be-37fa9e150e3a.png">

Also I just released version 1.5.0 and had no issues.

**Comment by jeffpaul** [comment]
@leonardola @LCmry so is there something that 10up should look to help with here to ensure PayFast works as desired with Woorelease?

**Comment by LCmry** [comment]
@jeffpaul I don't think so. This seems to have been an old issue that is not a problem anymore and the recent release did not surface any other issues.
